tools/generator/src/Generator/SemanticVersion.swift

```diff
@@ -45,21 +47,25 @@ struct SemanticVersion: Equatable, Hashable {
 
 extension SemanticVersion: Comparable {
     public static func < (lhs: SemanticVersion, rhs: SemanticVersion) -> Bool {
-        return (lhs.major < rhs.major)
-            || (lhs.major == rhs.major && lhs.minor < rhs.minor)
-            || (lhs.major == rhs.major && lhs.minor == rhs.minor && lhs.patch < rhs.patch)
+        guard lhs.major == rhs.major else {
```

Comment:
This can be done more simply:

```swift
(lhs.major, lhs.minor, lhs.patch) < (rhs.major, rhs.minor, rhs.patch)
```

Reply:
Me and my old pre-SE-0283 ways! Thanks 😄.